test/fw/ptl/utils/pbs_testsuite.py

```diff
@@ -597,6 +606,28 @@ def run_only_on_linux(cls):
                     test_item.__unittest_skip__ = True
                     test_item.__unittest_skip_why__ = msg
 
+    @classmethod
+    def check_mom_bash_version(cls):
+        skip_test = False
+        msg = 'capability supported only for bash version >= 4.2.46'
+        for mom in cls.moms.values():
+            if mom.check_mom_bash_version():
+                skip_test = True
+                break
+        if skip_test:
```

Comment:
This variable name might confuse. As I understand skip_test = True, then test will not skip. Right ? Then can we rename skip_test to no_skip(or something similar) or just swap the values default skip_test = True in loop update it to False and then use it in if condition?

Reply:
done